edb/schema/objtypes.py

```diff
@@ -125,8 +125,8 @@ def is_free_object_type(self, schema: s_schema.Schema) -> bool:
 
     def is_fake_object_type(self, schema: s_schema.Schema) -> bool:
         return (
-            self.get_name(schema).module == 'cfg'
-            or self.is_free_object_type(schema)
+            # self.get_name(schema).module == 'cfg' or
```

Comment:
Is this intended?

Reply:
Well, it should have been deleted, not commented out, but yeah